src/components/primitive/generic-modal/generic-modal.component.tsx

```diff
@@ -55,9 +61,27 @@ export const GenericModal: React.FunctionComponent<GenericModalComponentProps> =
         {!hide && (
           <Box row rightAlign paddingTop={2}>
             {!onCancel && (
-              <ModalButton label={{ text: 'Close', color: 'black' }} variant='ghost' onClick={handleClose} />
+              <ModalButton
+                label={{ text: "Close", color: "black" }}
+                variant="ghost"
+                onClick={handleClose}
+              />
+            )}
+            {!!onCancel && (
+              // @ts-ignore
+              <ModalButton {...onCancel} />
+            )}
+            {!onSubmit && (
+              <ModalButton
+                label={{ text: "Continue" }}
+                variant="small"
+                onClick={handleClose}
+              />
+            )}
+            {!!onSubmit && (
+              // @ts-ignore
+              <ModalButton {...onSubmit} />
             )}
```

Comment:
There is something wrong with the button, we can ignore this for now. I will open a new issue. will replace this later.